python/src/main/api.py

```diff
@@ -0,0 +1,31 @@
+from flask import Flask, jsonify, abort
+
+from python.src.main.parser import create_person, create_item
+
+app = Flask(__name__)
+
+
+class Api(object):
+    def __init__(self, db):
+        self.db = db
+
```

Comment:
This is where we define the endpoints, see below it looks like the end of a url. These are what we will used to communicate between the front end and the back end.

Reply:
We will communicate between the front end and back end using HTTP (provided by flask) with the data content in JSON format
https://www.json.org/
This is a very common format